appendices/symbol-lookup-cheatsheet.md

```diff
@@ -17,3 +17,55 @@ what you should search the tutorial for in order to learn more about the symbol.
 | ?  | Partial function |
 | '  | Prime |
 | <: | Subtype |
+
+
+Here is a more elaborate explanation of Pony's use of special characters:
+
+|Symbol | Usage|
+| --- | --- |
+| ,  | to separate parameters in a function signature, or the items of a tuple
+| .  | (1) to call a field or a function on a variable (field access or method call)
+     | (2) to qualify a type/method with its package name
```

Comment:
When there is a second line with (2), or sometimes a third line with (3), this means a 2nd or 3rd alternate usage of that symbol.

If it looks better, I could repeat the symbol.

Or I could add an explanation of (2) and (3).

Which seems better ?

 

From: Sean T Allen [mailto:notifications@github.com] 
Sent: maandag 11 april 2016 14:28
To: ponylang/pony-tutorial <pony-tutorial@noreply.github.com>
Cc: ibalbaert <ivo.balbaert@telenet.be>
Subject: Re: [ponylang/pony-tutorial] Added 2nd usage section for symbol cheat-sheet (#91)

 

In appendices/symbol-lookup-cheatsheet.md <https://github.com/ponylang/pony-tutorial/pull/91#discussion_r59195237> :

> @@ -17,3 +17,55 @@ what you should search the tutorial for in order to learn more about the symbol.
>  | ?  | Partial function |
>  | '  | Prime |
>  | <: | Subtype |
> +
> +
> +Here is a more elaborate explanation of Pony's use of special characters:
> +
> +|Symbol | Usage|
> +| --- | --- |
> +| ,  | to separate parameters in a function signature, or the items of a tuple
> +| .  | (1) to call a field or a function on a variable (field access or method call)
> +     | (2) to qualify a type/method with its package name

Seems to be missing | and symbol.

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub <https://github.com/ponylang/pony-tutorial/pull/91/files/80196091b4e62a5b9b923bb290ab4aac68909f35#r59195237>   <https://github.com/notifications/beacon/AA6zfAA2uWSHBMHXnhSGcsmVGk-bh6A8ks5p2j5EgaJpZM4IEQOw.gif> 



---
Dit e-mailbericht is gecontroleerd op virussen met Avast antivirussoftware.
https://www.avast.com/antivirus